Apps/DebugApp/DebugApp/DebugViewController.swift

```diff
@@ -77,8 +77,17 @@ public class DebugViewController: UIViewController {
          map and ensures that these layers would only be shown after the map has
          been fully rendered.
          */
-        mapView.on(.mapLoaded) { (event) in
+        mapView.on(.mapLoaded) { [weak self] (event) in
             print("The map has finished loading... Event = \(event)")
+            guard let self = self else { return }
+
+            try! self.mapView.__map.setStyleLayerPropertyForLayerId("country-label", property: "text-field", value: "MY_COUNTRY")
+
+            do {
+                _ = try self.mapView.style.getLayer(with: "country-label", type: SymbolLayer.self).get()
+            } catch {
+                print(error)
+            }
```

Comment:
Was it intentional to commit this?

Reply:
yes, I'll be removing once finished with all the commits